scripts/word-to-dtbook/src/main/csharp/CheckWordArchitecture/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.Win32;
```

Comment:
Are the csharp files still needed?

Reply:
Yes they are, this is the code of the shapes exporter for windows with Word.

Reply:
OK got it.